src/get-started/java-binding.md

```diff
@@ -604,13 +613,14 @@ The TestKit also allows creating blocks that contain all current [in-pool][in-po
 transactions:
 
 ```java
-// Instantiate a service that submits a transaction in it's `afterCommit` method
+// Instantiate a service that submits a transaction pool txs, e.g., submitted
```

Comment:
What does 'submits a transaction pool txs' mean?

Reply:
Rephrased that to `Instantiate a service that submits a transaction in its 'afterCommit' method. Such transactions are placed into the transaction pool`